### PR TITLE
init nestjs and vuejs projects

### PR DESCRIPTION
**Pull request by MelnychenkoIvan**
init nestjs and vuejs projects

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "extreme-sturtup",
```
Comment (KITSAnatoliiKarelov) [inline client/package.json]:
```suggestion
  "name": "extreme-startup" || "questionator",
```

Comment (KITSAnatoliiKarelov) [reply client/package.json]: (одне з двох)

File: client/public/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+    <title>extreme-sturtup</title>
```
Comment (KITSAnatoliiKarelov) [inline client/public/index.html]: Тут теж або 'extreme-startup' або 'questionator'

File: client/public/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  </head>
+  <body>
+    <noscript>
+      <strong>We're sorry but extreme-sturtup doesn't work properly without JavaScript enabled. Please enable it to continue.</strong>
```
Comment (KITSAnatoliiKarelov) [inline client/public/index.html]: Знову 'sturtup' <__<

File: package.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "questionator",
+  "version": "0.0.0",
+  "description": "extreme-sturtup",
```
Comment (KITSAnatoliiKarelov) [inline package.json]: See above

File: client/package.json (hunk 1 above)
Comment (MelnychenkoIvan) [reply client/package.json]: renamed in questionator

File: client/public/index.html (hunk 2 above)
Comment (MelnychenkoIvan) [reply client/public/index.html]: renamed in questionator

File: client/public/index.html (hunk 3 above)
Comment (MelnychenkoIvan) [reply client/public/index.html]: renamed in questionator

File: package.json (hunk 4 above)
Comment (MelnychenkoIvan) [reply package.json]: renamed in questionator

**Pull request merged**